server/models/Convo.js

```diff
@@ -0,0 +1,12 @@
+let mongoose = require("mongoose");
+
+const Convo = new mongoose.Schema({
+    users: [mongoose.ObjectId],
+    messages: {
```

Comment:
```suggestion
    messages: [Messages]
```
in my opinion you can do it like this.

Reply:
I've tried that haha what I meant is that that doesn't work, unless Messages is contained entirely within the same file.

Reply:
Change the export in Message model.

Reply:
The error that I keep getting is related to the import in Convo.js:
Cannot use import statement outside a module

Reply:
Message me on slack and share a photo of Convo.js and Message.js model

Reply:
I think you addressed in a separate comment 

```
messages: {    type: mongoose.Schema.Types.ObjectId,    required: true,    ref: "Message",  },
```